Lesson10_Task01.py

```diff
@@ -0,0 +1,29 @@
+"""
+Home-task Regex
+
+# open input.txt file and find all small
+ english letters that match such conditions:
+# target small letter round exactly with
+ three capital english letters on the left and on the right
+# Match examples:
+# sdTRYaUVKn  -> matches "a"
+# NTRSjARTb   -> no match ( not exactly 3 capital letters on the left)
+# zDFGbOPNq   -> matches "b"
+# Print Output as : "Result: "<your_found_human_readable_string">
+"""
+
+import re
+
+# Read the input file
+with open("input02.txt", "r") as input_file:
+    # Read the contents of the file
+    content = input_file.read()
+
+# Find all small English letters that match the specified conditions
+matches = re.findall(r'(?<=[A-Z]{3})([a-z])(?=[A-Z]{3})', content)
```

Comment:
Unfortunately incorrect solution. Please pay attention that
zDFGbOPNq -> matches "b"
PDFGbOPNqa - > no match cause left side have 4 capital letters instead of expected 3.

So you need additionally somehow define that before 3 capital letters should be something non capital + behind ( right part) 3 capital letters should be something non capital.